docker-compose.yml

```
@@ -51,6 +51,20 @@ services:
       retries: 10
       timeout: 5s
       start_period: 30s
+  oasis-net-runner:
+    image: oasisprotocol/oasis-net-runner:dev
+    volumes:
+      - type: bind
+        source: ./
+        target: /oasis-indexer
+    command:
+      - /bin/sh
+      - -c
+      - oasis-net-runner --basedir.no_temp_dir --basedir /testnet --fixture.default.node.binary oasis-node --fixture.default.setup_runtimes=false --fixture.default.deterministic_entities=true --fixture.default.fund_entities --fixture.default.num_entities 2
```

```suggestion
      - oasis-net-runner
       - --basedir.no_temp_dir
       - --basedir /testnet
       - --fixture.default.node.binary oasis-node
       - --fixture.default.setup_runtimes=false
       - --fixture.default.deterministic_entities=true
       - --fixture.default.fund_entities
       - --fixture.default.num_entities 2
```
nit but might help with readability/extensibility, the line is already getting unwieldy long

---

Oops I didn't intend to include this file; it's a leftover from my testing setup. Removing